timelock-agent/src/main/java/com/palantir/timelock/config/PaxosRuntimeConfiguration.java

```diff
@@ -33,6 +33,12 @@ default long leaderPingResponseWaitMs() {
         return 5000L;
     }
 
+    @JsonProperty("only-log-on-quorum-failure")
```

Comment:
Why do we need this on both `PaxosRuntimeConfiguration` and `LeaderRuntimeConfig`?  We could just pass the `PaxosRuntimeConfiguration` around everywhere or even just an anonymous `Supplier<Boolean>`.  If we do stick with `LeaderRuntimeConfig` then I don't think it needs the JSON serializable stuff.

Reply:
Removing the JSON serializable stuff. I am also planning to add paxos ping rates to this class. Thought would be tidier this way.